blueocean-core-js/package.json

```diff
@@ -54,6 +53,7 @@
     "react-dom": "^15.1.0"
   },
   "devDependencies": {
+    "@jenkins-cd/design-language": "0.0.123-unpublished-jm-b",
```

Comment:
It's published, but not under the "latest" production tag. Named as such because it's a variation of ".123-unpublished" which is JDLrish for "-SNAPSHOT"